README.md

```diff
@@ -5,14 +5,37 @@ Just a simple pecan app
 
 1. clone the repo
 
-2. install pecan in virtual env: https://pecan.readthedocs.io/en/latest/installation.html
+2. cd flocx-market and run cmd:
+```
+pip install -r requirements.txt
+```
+3. migrate the database:
+```
+cd flocx_market/db/sqlalchemy
+```
+```
+alembic upgrade head
```

Comment:
At some point (I guess it doesn't have to be this PR) we will want a command line tool in this project that takes care of database initialization/upgrades. The tool should be installed as part of the `pip install` operation. See for example the [esi-leap](https://github.com/cci-moc/esi-leap) project, which installs an `esi-leap-dbsync` command: https://github.com/CCI-MOC/esi-leap/blob/master/setup.cfg#L17